desktop/src/app/components/gallery/home/gallery.component.ts

```diff
@@ -32,6 +32,8 @@ export class GalleryComponent implements OnInit, OnDestroy {
     private _baseUrl: string;
     private _destroy = new Subject();
 
+    public deprecateRenderingDate = new Date(2024, 1, 29).toDateString()
```

Comment:
Can we use a localized date string here? If we don't have any good utility functions for this, we may want to add something to bonito-core